recipes/libnghttp3/all/conanfile.py

```diff
@@ -0,0 +1,88 @@
+from conan import ConanFile
+from conan.errors import ConanInvalidConfiguration
+from conan.tools.apple import is_apple_os
+from conan.tools.cmake import CMake, CMakeToolchain, CMakeDeps, cmake_layout
+from conan.tools.files import get, rmdir, copy
+from conan.tools.gnu import PkgConfigDeps
+from conan.tools.microsoft import is_msvc
+from conan.tools.scm import Version
+import os
+
+
+required_conan_version = ">=1.53.0"
+
+
+class Nghttp3Conan(ConanFile):
+    name = "libnghttp3"
```

Comment:
```suggestion
    name = "nghttp3"
```
Please, keep the project's name to be consistent. That means you will need to change the folder name too.

Reply:
I'm going to make it consistent with libnghttp2 by changing everything to libnghttp3 and removing nghttp3 references. Libnghttp2 has these inconsistencies, but at the end of the day, that recipe lives in a folder of that name and produces a package of that name.

Reply:
The libnghttp2 was not well reviewed and was long time ago, it should be `nghttp2` instead. In the very past we had a wrong idea of creating separated packages for binaries and libraries. 

As you can see, the repology website finds many references to https://repology.org/project/nghttp2/information and same for nghttp3.

Please, let's do it right now, rename it `nghttp3`.